cachito/web/models.py

```diff
@@ -62,6 +62,29 @@
 )
 
 
+class ContentManifest:
+    """A content manifest associated with the request."""
+
+    version = 1
+    json_schema_url = (
+        "https://raw.githubusercontent.com/containerbuildsystem/atomic-reactor/"
+        "f4abcfdaf8247a6b074f94fa84f3846f82d781c6/atomic_reactor/schemas/content_manifest.json"
```

Comment:
Using a specific commit hash seems brittle...

Reply:
Could you use a Git tag instead of a commit hash?

Reply:
The reason I used the commit in the first place was due to the floating nature of tags. I can move to a tag, knowing we do control the tags in atomic-reactor and they will not change.

Reply:
Actually, we do not have an atomic-reactor release with it, so we would need to wait for the next release cycle to change this. Any other suggestions?

Reply:
@athos-ribeiro that's fine. Let's just update it once it's released or you can set it now knowing the URL is invalid until the release.